src/wallet/wallet.h

```diff
@@ -735,7 +735,7 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * Insert additional inputs into the transaction by
      * calling CreateTransaction();
      */
-    bool FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, int& nChangePosRet, std::string& strFailReason, bool includeWatching);
```

Comment:
I guess we should rename `nChangePosRet ` to `nChangePosInOut` and add a comment in the header that -1 as a input value will result in setting a random position.